### DISCUSSION TITLE
Any demo repositories which has client side script for colyseus unity?

### DISCUSSION THREAD

**Discussion by Nahidma007** [colyseus/colyseus#601] [Q&A]
Are there any demo repositories which has client side script for colyseus unity?. Initially I have done my server side in node.js and client side in javascript. I wanted to migrate to unity so had confusion related with schema. In javascript there was no need to define schema in client side but in unity client side, we have to define schema. Any solution about how to write in client side of unity because I am quite new to unity. 
If there any repos, please consider to share

**Comment by endel** [comment]
Hi @Nahidma007, in Unity it is required to generate the schema definitions for the client-side using `schema-codegen`: https://docs.colyseus.io/state/schema-callbacks/#client-side-schema-generation

This example uses version 0.14 but code can still be used as reference for 0.15: https://github.com/colyseus/unity-demo-tanks
